cmd/dagger/module.go

```diff
@@ -59,6 +59,26 @@ const (
 	defaultModuleSourceDirName = "."
 )
 
+// TODO(rajatjindal): are there any other specific files
+// that we want to ignore
+func getDefaultSourcePathDir(srcRootPath string) (string, error) {
+	list, err := os.ReadDir(srcRootPath)
+	if err != nil {
+		return "", err
+	}
+
+	for _, l := range list {
+		// ignore hidden files
+		if strings.HasPrefix(l.Name(), ".") {
+			continue
```

Comment:
are there any specific dirs that we would like to ignore when checking if the current folder has files/folders. Right now it ignores only the hidden folders.

Reply:
Agree with Solomon that we don't need to check anything extra for now.

However, we probably *should* add an exception for `.dagger` here. Obscure, but if the only thing we find is `.dagger` it probably still makes sense to use it? I could see this mattering in cases where someone is trying to "re-init" by deleting everything in `.dagger` but leaves that dir itself around.

Reply:
Hi @sipsma, there are a few scenarios here:

1. `.dagger` exists and no other file exists - currently it will use current dir. I've pushed a change to use .dagger in this scenario.
2. `.dagger` exists and other files exists in the current folder - will already use .dagger
3. `.dagger` does not exist and other files exists - will already use .dagger
4. blank dir - will use current dir

thanks